Bottom: f63a0cba6a54c698b2ccb223019d4ffa106360c9
Top:    8d3456f0d740c4877b3427b598d635a53dbc420b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 16:59:14 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 4edb048..99d243a 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2460,7 +2460,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
